### PR TITLE
Foundations - Global styles

### PR DESCRIPTION
**Pull request by Brayan1425**
add global styles and paginations.

### CODE REVIEW

**Review by Theweeksad — APPROVED** [review]
very good, you changed the readme

File: main.css [hunk 1]
```diff
@@ -1 +1,34 @@
...
+@font-face {
+    font-family: Nunito;
+    src: url(./asset/typography/Nunito-Regular.ttf);
+}
+
+h1 {
+    font-family:Noto-Serif ;
+    font-size: 50px;
+    text-align: center;
+
```
Comment (francoo-trainer) [inline main.css]: avoid making line break

File: main.css [hunk 2]
```diff
@@ -1 +1,34 @@
...
+h2 {
+    font-family:Nunito ;
+    font-size:25px ;
+    margin-left: 30px;
+}
+
+p {
+    font-family:Nunito ;
+    margin-left: 30px;
+
```
Comment (francoo-trainer) [inline main.css]: avoid making line break

**Comment by francoo-trainer** [comment]
nice! 
avoid making line break

**Review by Jasoneto — CHANGES_REQUESTED** [review]
Changes approved.

**Review by johanmendezb — APPROVED** [review]

**Review by Jasoneto — APPROVED** [review]
the ticket is already done.

File: main.css (hunk 1 above)
Comment (Brayan1425) [reply main.css]: fixed, thank you very much

File: main.css (hunk 2 above)
Comment (Brayan1425) [reply main.css]: fixed, thank you very much

**Review by francoo-trainer — APPROVED** [review]

**Pull request merged**